code/game/objects/structures/flora.dm

```diff
@@ -1,204 +1,320 @@
-//trees
+////////////////////////////////////////
+// Base Flora
+////////////////////////////////////////
+/obj/structure/flora
+	desc                   = "A form of vegetation."
+	anchored               = TRUE
+	density                = FALSE                                  //Plants usually have no collisions
+	w_class                = ITEM_SIZE_NORMAL                       //Size determines material yield
+	material               = /decl/material/solid/plantmatter       //Generic plantstuff
+	tool_interaction_flags = 0 
+	hitsound               = 'sound/effects/hit_bush.ogg'
+	var/tmp/snd_cut        = 'sound/effects/plants/brush_leaves.ogg' //Sound to play when cutting the plant down
+	var/remains_type       = /obj/effect/decal/cleanable/plant_bits //What does the plant leaves behind in addition to the materials its made out of. (part_type is like this, but it drops instead of materials)
+
+/obj/structure/flora/Initialize(ml, _mat, _reinf_mat)
+	. = ..()
+	init_appearance()
```

Comment:
Is this separate from Initialize() for serialization purposes downstream? Seems like an odd change otherwise.